lms/templates/certificates/_accomplishment-rendering.html

```diff
@@ -68,14 +69,14 @@ <h3 class="accomplishment-orgs-title sr-only">${_("Supported by the following or
                 <ul class="wrapper-orgs list-orgs">
                     <li class="wrapper-organization">
                         <div class="organization organization-edX">
-                            <img class="organization-logo" src="${logo_src}" alt="${platform_name}">
+                            <img class="organization-logo" src="${logo_src | n, decode.utf8}" alt="${platform_name}">
                         </div>
                     </li>
 
                     % if organization_logo:
                     <li class="wrapper-organization">
                         <div class="organization">
-                            <img class="organization-logo" src="${organization_logo.url}" alt="${platform_name}">
+                            <img class="organization-logo" src="${organization_logo.url | n, decode.utf8}" alt="${platform_name}">
```

Comment:
We still need to apply HTML escaping to the img source URL. Please remove the n filter. See http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/conventions/preventing_xss.html#url-context-and-escaping for more details.